src/test/java/teammates/logic/core/InstructorsLogicTest.java

```diff
@@ -317,6 +322,58 @@ private void testVerifyAtLeastOneInstructorIsDisplayed() throws Exception {
                         true, true));
     }
 
+    @Test
+    public void testUpdateInstructorByGoogleIdCascade_shouldCascadeUpdateToFeedbackSessions() throws Exception {
+        InstructorAttributes instructorToBeUpdated = dataBundle.instructors.get("instructor1OfCourse1");
+        String courseId = instructorToBeUpdated.getCourseId();
+        String oldEmailAddress = instructorToBeUpdated.getEmail();
+        String newEmailAddress = "new@email.tmt";
+
+        Map<Instant, Integer> oldDeadlineCounts = fsLogic.getFeedbackSessionsForCourse(courseId)
+                .stream()
+                .map(FeedbackSessionAttributes::getInstructorDeadlines)
+                .filter(instructorDeadlines -> instructorDeadlines.containsKey(oldEmailAddress))
+                .map(instructorDeadlines -> instructorDeadlines.get(oldEmailAddress))
+                .reduce(new HashMap<>(), (counts, deadline) -> {
```

Comment:
A few more places like this are unclear. Can apply the same fix here

Reply:
Ah sorry, I forgot I did it in other files too.